features/step_matches_message.feature

```diff
@@ -81,3 +81,21 @@ Feature: Step matches message
 
       """
     And the stderr should not contain anything
+
+  Scenario: Step matches initialize step definitions compatible with html formatter
+
+    Given there is a wire server running on port 54321 which understands the following protocol:
+      | request                                              | response                                                                           |
+      | ["step_matches",{"name_to_match":"we're all wired"}] | ["success",[{"id":"1", "args":[], "source":"MyApp.MyClass:123", "regexp":"we.*"}]] |
+      | ["begin_scenario"]                                   | ["success"]                                                                        |
+      | ["invoke",{"id":"1","args":[]}]                      | ["success"]                                                                        |
+      | ["end_scenario"]                                     | ["success"]                                                                        |
+    When I run `cucumber --format message --out messages.json --publish-quiet`
+    Then it should pass with:
```

Comment:
Should we have an additional assertion on the JSON output. You can just JSON.load the property to get a hash like structure?

Reply:
So, while working to add a validation of the json itself as requested, I noticed that the "TestRunFinished" message was always returning `success: false`.
I've opened an issue in cucumber-ruby: https://github.com/cucumber/cucumber-ruby/issues/1604

So, what about adding a test wich will pass for now, then fix it once it will broke after we would have fix https://github.com/cucumber/cucumber-ruby/issues/1604?

Reply:
Bet you love me coming back after Xmas right... :)

I'm happy with whichever approach you think is best. If we can sign off this PR AS IS. i.e. just add in a rudimentary always passing test, that will later break, that would be my preferred approach

Reply:
> Bet you love me coming back after Xmas right... :)

Actually, yes! I was waiting for you 😄
Hope you feel good!



> If we can sign off this PR AS IS. i.e. just add in a rudimentary always passing test, that will later break, that would be my preferred approach

looks like a plan 👍